app/src/main/java/com/hillnerds/soundsparrow/SoundGeneration.java

```diff
@@ -10,135 +18,88 @@
 
 public class SoundGeneration {
```

Comment:
I think that the distinction between SoundGeneration and ChannelSoundGeneration is important as the first one takes care of everything to do with mergin and sorting MIDI sequence files and the sedond is responsible for generation of MidiSequences for one Channel Specifically.